app.rb

```diff
@@ -10,6 +11,66 @@ def initialize
     @rentals = []
   end
 
+  # rubocop:disable Metrics/PerceivedComplexity
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def read_data
+    File.new('Data/people.json', 'w') unless File.exist?('Data/people.json')
+    File.new('Data/books.json', 'w') unless File.exist?('Data/books.json')
+    File.new('Data/rentals.json', 'w') unless File.exist?('Data/rentals.json')
+    people_json = File.read('Data/people.json')
+    people = JSON.parse(people_json)
+    people.each do |person|
+      if person['type'] == 'student'
+        @people.push(Student.new(person['age'], person['name'], person['parent_permission']))
+      elsif person['type'] == 'teacher'
+        @people.push(Teacher.new(person['age'], person['specialization'], person['name']))
+      end
+    end
+    books_json = File.read('Data/books.json')
+    books = JSON.parse(books_json)
+    books.each do |book|
+      @books.push(Book.new(book['title'], book['author']))
+    end
+  end
```

Comment:
- [ ] **Metrics/PerceivedComplexity and Metrics/CyclomaticComplexity:**
Instead of disabling these metrics for the entire method, please try to refactor the code to simplify it and reduce complexity as linters help you to adhere to coding practices. Therefore it isn't advisable to disable them 👍 
- [ ] Please handle the case when one or more of the files are missing (the app should work and show no errors). The error below is thrown when I delete your data and try to start the application  👇 

![image](https://github.com/Othniel2471/school-library/assets/46839548/91748574-6fcf-4f6b-9905-574621812150)